server.go

```diff
@@ -99,6 +106,23 @@ type Server struct {
 	enableProfiling bool
 
 	HistogramAggregates samplers.HistogramAggregates
+
+	// TODO we don't want to hardcode this
+	// TODO we need to actually configure lightstep
+	// TODO also accessing it is a race condition
+	tracerSinks []tracerSink
```

Comment:
To be honest, all of our configuration options are racy in that sense, so I'm going to remove the TODO. I can't conceive of a situation under which we'd want to add more sinks at a time that *isn't* initialization.